gitcoinco/grants-stack
# implement lit encryption and decryption for allo v2

Fixes: #issue

## Description

how to test:

- create a round on v2 with and encrypted field (do not use sepolia)
- apply from builder
- go back to manager and check that the field is decrypted when viewing the application


## Checklist

This PR:

- [ ] Does it add new payout or donation token? In this case, have they been added to the indexer to avoid it to crash?
